### ISSUE TITLE
Better documentation

### ISSUE THREAD

**Issue by dbautistav** [dbautistav/XAUFI#22]
@jbarbosat: puedes describir tu propuesta de documentación para que esta herramienta sea más útil, por favor? Gracias!

**Comment by jbarbosat** [comment]
Le moví una línea al readme en master. Chance un índice de qué hay en cada carpeta sirva? lo agrego al readme de 16.0.4 y me dices q te parece?

**Comment by dbautistav** [comment]
Sí, haz los cambios que consideres necesarios. Si tienes duda al hacer un cambio o quieres discutirlo primero, crea una rama nueva y ahí lo tratamos antes de mezclarlo a master. Te parece? Gracias!

**Comment by jbarbosat** [comment]
Sale, ya le agregué un índice a los readme's de cada branch para ver q scripts hay
